apps/web/src/components/magazine-list/index.tsx

```diff
@@ -1,18 +1,18 @@
 import type {
+  Document,
   Magazine,
   MagazineIssue,
   Media,
-  Document,
 } from "@tietokilta/cms-types/payload";
-import Image from "next/image";
 import _ from "lodash";
+import Image from "next/image";
 import TikLogo from "../../assets/TiK-logo.png";
 
 function IssueCard({ issue }: { issue: MagazineIssue }) {
   const thumbnail = issue.thumbnail as Media;
   const file = issue.file as Document;
   return (
-    <div className="w-full max-w-xs sm:w-44">
+    <div className="w-full max-w-xs min-[460px]:w-44">
```

Comment:
Not a big deal, but it would be nice if we could make it properly responsive without needing to add a new (arbitrary) breakpoint

Reply:
It would be, but I don't know how to do that :sweat_smile:

Reply:
I can take a look at another time, but what exactly is the issue with the current layout? I was checking it out and it seems to work mostly fine. I guess on mobile screen sizes it's kind of annoying how big the single magazines are.

It might be helpful to take a look at the committee members tables, I made some layout improvements there especially targeted at small screens recently and it looks mostly okay now

Reply:
One issue is that sometimes there is extra space on the right:
![image](https://github.com/Tietokilta/web/assets/74378567/0fae4854-de9d-47cd-8131-406371ac4f38)
Of course there are surely other ways to fix that, but it was fixed when I changed flexbox to grid because I wanted to restrict maximum number of issues per row to four. Only problem with that was how to change the number of columns depending on the screen size which was why I used those arbitrary breakpoints.

Reply:
If this "3 per line " thing is something you'd want to achieve, CSS Grid with some templaring might make sense here I guess. Not that I know how to use it, but I figure it's what you'd want 😅

Reply:
Ah, yeah it should be really easy then with grid. If you just remove the rigid width from the items and fill all space, and only define the column count with breakpoints it should work perfectly.

`grid grid-cols-1 gap-8 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4`